docs/source/onedal/algorithms/kernel-functions/index.rst

```diff
@@ -18,6 +18,37 @@
 Kernel Functions
 ================
 
+A kernel function takes input vectors from original (:math:`p`-dimensional) space
+and returns dot product of the vectors in :math:`s`-dimensional feature space.
+More formally, if we have :math:`x,y \in \mathbb{R}^p`, and
```

Comment:
```suggestion
For example, if you have :math:`x,y \in \mathbb{R}^p`, and
```

Reply:
This is not an example, but a mathematical definition, which was described with words above. 
So changed to
`Thus, having :math:....`